### ISSUE TITLE
Publish OSMD to Bower

### ISSUE THREAD

**Issue by sebastianhaas** [opensheetmusicdisplay/opensheetmusicdisplay#12]
Requires #11 to be finished first.
Publish to bower for easy use via script tags.

**Comment by sebastianhaas** [comment]
Can be published after #58

**Comment by sebastianhaas** [comment]
Did some research today, it appears to me that Bower moved away from _just_ providing a searchable repository for JS libraries. They don't want minified libraries to be published, but rather JS, CSS etc. source files along with dependencies specified so other people can include it in their build systems.

That is, for now, out of scope for opensheetmusicdisplay, since we already provide a complete bundle, including all dependencies. Bower would just make this more complicated than it actually is.

My suggestion:
* Keep `npm install opensheetmusicdisplay`  as the preferred way to use it as a developer who wants to integrate deeply with opensheetmusicdisplay.
* Publish the bundled version somewhere for easy use via script tags, so that people can put ```html
<script src="<location>/opensheetmusicdisplay.js" type="text/javascript">
```
 * Self-hosted on AWS S3 for a start
 * http://www.jsdelivr.com/
 * https://cdnjs.com/

The latter require a request for hosting open source libraries. I don't think it makes sense to do this before we have a non-pre-release, so please @bukaznik file an issue and schedule this for `0.1.0`.
Also cc @petrosalema would love to hear your input on this, since I'm not really up to date on  frontend-related stuff.
